services/scabbard/libscabbard/src/service/factory/factory_database.rs

```diff
@@ -44,9 +44,6 @@ use crate::store::{
 
 const DEFAULT_LMDB_DIR: &str = "/var/lib/splinter";
 
-#[cfg(feature = "receipt-store")]
-type ScabbardReceiptStore = Arc<RwLock<dyn ReceiptStore>>;
```

Comment:
This change looks unrelated to removing the feature.

Reply:
I removed it because the type was originally created so that separate args weren't needed for the two different receipt stores. It was also removed from `factory_static_configuration.rs` but you're right, in this file it isn't directly related so I changed this back